docs/porting/psa/spm.md

```diff
@@ -0,0 +1,257 @@
+<h2 id="spm-port">PSA SPM</h2>
+
+Secure Partition Manager (SPM) is a part of the PSA Firmware Framework that is responsible for isolating software in partitions, managing the execution of software within partitions and providing interprocessor communication (IPC) between partitions.
+
+For more information about SPM, please refer to [the SPM overview page](/docs/development/apis/mbed-psa.html).
+
+**This page gives guidelines for silicon partners wishing to have Secure Partition Manager capabilities**
+
+### Memory layout

```

Comment:
@alzix:
Since silicon partners have to add a new target to targets.json, I think the section headline should be called targets.json editing (or something lie that).
In this section we will describe the targets.json required modification.
Memory addresses and sizes (the "overrides" part) are a part of this, and memory layout explanation can be a sub-section here.
Except for "overrides" and the "inherits" parts (which you have mentioned) we should mention:
- "extra_labels_add" part
- "components_add" part (mailbox)
- "deliver_to_target"
What do you think?